main.go

```diff
@@ -28,7 +31,17 @@ func main() {
 	if *showJSON {
 		showModuleJSON(module)
 	} else {
-		showModuleMarkdown(module)
+		var markdownTemplate = defaultMarkdownTemplate
+
+		if len(*templateFile) > 0 {
+			rawTemplate, err := ioutil.ReadFile(*templateFile)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
Instead of `log.Fatal`, let's stick with the convention used in the rest of this code:

```
fmt.Fprintf(os.Stderr, "error reading template: %s\n", err)
os.Exit(2)
```